scripts/unit-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+uv python find 3.10.16
+IS_PYTHON=$?
+if [ $IS_PYTHON -ne 0 ]; then
+     uv python install 3.10.16
+fi
+uv run -p 3.10.16 --with-editable . --with-editable ".[dev]" --with-editable ".[unit]" pytest --cov=llama_stack -s -v tests/unit/ --junitxml=pytest-report.xml
```

Comment:
output-ing coverage during development seems to just show a giant wall of text. shouldn't we keep that limited to CI, or have an option for this?